javacord-api/src/main/java/org/javacord/api/BotInviteBuilder.java

```diff
@@ -12,7 +12,7 @@
      * The base link of a bot invite.
      */
     public static final String BASE_LINK =
-            "https://discordapp.com/oauth2/authorize?client_id=%s&scope=bot&permissions=%s";
+            "https://discord.com/oauth2/authorize?client_id=%s&scope=bot&permissions=%s";
```

Comment:
When we have the constants, we should also use them.
```suggestion
            "https://" + DiscordApi.DISCORD_DOMAIN + "/oauth2/authorize?client_id=%s&scope=bot&permissions=%s";
```